diff --git a/node_modules/@perfectmemory/ngx-contextmenu/fesm2022/perfectmemory-ngx-contextmenu.mjs b/node_modules/@perfectmemory/ngx-contextmenu/fesm2022/perfectmemory-ngx-contextmenu.mjs
index 78c1246..60f962a 100644
--- a/node_modules/@perfectmemory/ngx-contextmenu/fesm2022/perfectmemory-ngx-contextmenu.mjs
+++ b/node_modules/@perfectmemory/ngx-contextmenu/fesm2022/perfectmemory-ngx-contextmenu.mjs
@@ -1,9 +1,9 @@
 import * as i1 from '@angular/cdk/overlay';
 import { OverlayModule, FullscreenOverlayContainer, OverlayContainer } from '@angular/cdk/overlay';
 import * as i2 from '@angular/common';
-import { CommonModule } from '@angular/common';
+import { CommonModule, DOCUMENT } from '@angular/common';
 import * as i0 from '@angular/core';
-import { EventEmitter, Output, Input, Optional, Directive, Injectable, DOCUMENT, HostListener, HostBinding, ViewChildren, Inject, ChangeDetectionStrategy, Component, ElementRef, ContentChildren, ViewEncapsulation, NgModule } from '@angular/core';
+import { EventEmitter, Output, Input, Optional, Directive, Injectable, HostListener, HostBinding, ViewChildren, Inject, ChangeDetectionStrategy, Component, ElementRef, ContentChildren, ViewEncapsulation, NgModule } from '@angular/core';
 import { FocusKeyManager } from '@angular/cdk/a11y';
 import { Subscription } from 'rxjs';
 import { ComponentPortal } from '@angular/cdk/portal';
