diff --git a/node_modules/angular-calendar/fesm2022/angular-calendar.mjs b/node_modules/angular-calendar/fesm2022/angular-calendar.mjs
index a725f56..347bd35 100644
--- a/node_modules/angular-calendar/fesm2022/angular-calendar.mjs
+++ b/node_modules/angular-calendar/fesm2022/angular-calendar.mjs
@@ -1,6 +1,6 @@
 import * as i0 from '@angular/core';
-import { EventEmitter, inject, Renderer2, ElementRef, Output, Input, Directive, NgZone, Injectable, LOCALE_ID, Pipe, Component, Injector, ViewContainerRef, DOCUMENT, HostListener, InjectionToken, NgModule, ChangeDetectorRef } from '@angular/core';
-import { formatDate, NgClass, NgTemplateOutlet, NgStyle, SlicePipe, AsyncPipe } from '@angular/common';
+import { EventEmitter, inject, Renderer2, ElementRef, Output, Input, Directive, NgZone, Injectable, LOCALE_ID, Pipe, Component, Injector, ViewContainerRef, HostListener, InjectionToken, NgModule, ChangeDetectorRef } from '@angular/core';
+import { formatDate, NgClass, NgTemplateOutlet, NgStyle, SlicePipe, AsyncPipe, DOCUMENT } from '@angular/common';
 import { Subject, Observable, of, timer, BehaviorSubject, interval } from 'rxjs';
 import { takeUntil, switchMap, startWith, map } from 'rxjs/operators';
 import { positionElements } from 'positioning';
